### PR TITLE
Store README assets in repository

### LINKED ISSUE #2140: Store assets in our repository rather than calling third party services
- Store assets in our repository rather than calling third party services, As it may tend to call too much third party services and leads to slow loading time.

- Also some of the assets are not loading

### PR DESCRIPTION
**Pull request by Nikhil-Vats**
Fixes #2140 

#### Short description of what this resolves:

Stores assets used in README in repository instead of rendering them from web.

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by kushthedude** [comment]
Update the `docker` svg too in installation docs

**Comment by Nikhil-Vats** [comment]
@kushthedude I have stored all the assets in installation docs locally except the docker svgs as I could not found them. I found similar queries on docker forums - see [this](https://forums.docker.com/t/deploy-to-docker-cloud-button-only-supported-in-standard-mode/29517/2). Even the docs ain't available anymore, they were available in beta version - https://beta-stage.docs.docker.com/v17.12/docker-cloud/apps/deploy-to-cloud-btn/. We can replace it with just text or docker logo svg. What do you suggest?

File: README.md [hunk 1]
```diff
@@ -2,10 +2,10 @@
 
 ![Badgeyay](/frontend/public/images/Badgeyay-artwork.png)
 
-[![Gitter](https://img.shields.io/badge/chat-on%20gitter-ff006f.svg?style=flat-square)](https://gitter.im/fossasia/badgeyay)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/1ac554483fac462797ffa5a8b9adf2fa?style=flat-square)](https://www.codacy.com/app/fossasia/badgeyay)
-[![Build Status](https://api.travis-ci.org/fossasia/badgeyay.svg?branch=development&style=flat-square)](https://travis-ci.org/fossasia/badgeyay)
```
Comment (kushthedude) [inline README.md:7]: Standard Badges should not be changes

Comment (Nikhil-Vats) [reply README.md:7]: I haven't changed the standard badges, I downloaded them from the URL mentioned and just changed their name, they are exactly same.

**Comment by Nikhil-Vats** [comment]
@kushthedude For now, I have replaced the broken docker svg with text, let me know if you want it to be replaced with **docker logo** or something else instead. I can also **create a custom button**. Take a look at the readme [**here**](https://github.com/Nikhil-Vats/badgeyay/tree/development/docs/backend/installation#installation).

**Comment by kushthedude** [comment]
No need, docker deploy is not active too.

**Pull request closed without merging**